kbs/config/kubernetes/README.md

```diff
@@ -147,6 +151,10 @@ kbs    ClusterIP   10.0.210.190   <none>        8080/TCP   4s
 
 ## Delete KBS
 
+```bash
+$ kubectl delete -k ${DEPLOYMENT_DIR}/
 ```
-$ kubectl delete -k ${DEPLOYMENT_DIR}/$(uname -m)
+or, if running on IBM Secure Execution run:
+```bash
+$ kubectl delete -k ${DEPLOYMENT_DIR}/ibm-se/
```

Comment:
Could you please add `$ kubectl delete pv test-local-pv` here?

Reply:
Should be done now.